geolocator_android/android/src/main/java/com/baseflow/geolocator/location/BackgroundNotification.java

```diff
@@ -88,6 +88,12 @@ private void updateNotification(
                 .setContentIntent(buildBringToFrontIntent())
                 .setOngoing(options.isSetOngoing());
 
+        int notificationColor = options.getColor();
+        if (notificationColor != -1) {
+            builder = builder
+                .setColor(options.getColor());
```

Comment:
Seems more logical to use the `notificationColor` here:

```suggestion
                .setColor(notificationColor);
```